app/build.gradle

```diff
@@ -29,7 +29,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.github.ibm-cloud-security:appid-clientsdk-android:6.+'
+    implementation 'com.github.ibm-cloud-security:appid-clientsdk-android:6.0.1'
```

Comment:
why we cant leave it 6.+
it supposed to get lates version which is 6.0.1

Reply:
I thought that if somehow it gets <6.0.1 we'll get an error that AppID.logout is undefined

Reply:
changed back to 6.+